Mathlib/RingTheory/Algebraic/Integral.lean

```diff
@@ -211,6 +211,15 @@ section trans
 variable (R) [NoZeroDivisors S] (inj : Function.Injective (algebraMap S A))
 include inj
 
+/- This does not hold if only `R` is assumed to be a domain but `S` is not: for example, let
```

Comment:
I'm guessing you didn't make this a docstring on purpose, but is this of enough interest to have it show up in the generated docs?
```suggestion
/-! 
The following does not hold if only `R` is assumed to be a domain but `S` is not: for example, let
```

Reply:
```suggestion
/-! 
The next theorem may fail if only `R` is assumed to be a domain but `S` is not: for example, let
```

Reply:
I changed the wording a bit. The example is maybe a bit too long, and I just want people who edit the file to see it so they won't make an attempt to generalize.